cpp/include/cudf/quantiles.hpp

```diff
@@ -63,21 +63,28 @@ quantile(column_view const& input,
  * @param input           Table used to compute quantile rows.
  * @param q               Desired quantiles in range [0, 1].
  * @param interp          Strategy used to select between the two rows on either
-                          side of the desired quantile.
- * @param sorted          Indicates if the input has been pre-sorted.
- * @param column_order    The desired sort order for each column.
- * @param null_precedence The desired order of null compared to other elements.
+ *                        side of the desired quantile.
+ * @param sortmap         Indicates the sorted order of the inputs. If empty,
```

Comment:
I find the name `sortmap` confusing. How about `ordered_indices`? Also need to document the required type.
```suggestion
 * @param sortmap         a column of integer indices that order `input`. If empty, `input` is assumed to be sorted.
```